include/crocoddyl/multibody/residuals/pair-collisions.hpp

```diff
@@ -0,0 +1,157 @@
+///////////////////////////////////////////////////////////////////////////////
+// BSD 3-Clause License
+//
+// Copyright (C) 2021, LAAS-CNRS, University of Edinburgh
+// Copyright note valid unless otherwise stated in individual files.
+// All rights reserved.
+///////////////////////////////////////////////////////////////////////////////
+
+#ifndef CROCODDYL_MULTIBODY_RESIDUALS_PAIR_COLLISIONS_HPP_
+#define CROCODDYL_MULTIBODY_RESIDUALS_PAIR_COLLISIONS_HPP_
+
+#include "crocoddyl/core/residual-base.hpp"
```

Comment:
Rearrange the header inclusions. As rule of thumb, you should always include external header first. So the code should be
```cpp
#include <pinocchio/multibody/geometry.hpp>
#include <pinocchio/algorithm/jacobian.hpp>
#include <pinocchio/algorithm/geometry.hpp>
#include "crocoddyl/multibody/states/multibody.hpp"
#include "crocoddyl/multibody/data/multibody.hpp"
#include "crocoddyl/core/utils/exception.hpp"
```

Note that I use `<>` for external headers as we follow the Google code style (and it is a good practice).